specviz/data/ui/plot_window.ui

```diff
@@ -71,6 +71,7 @@
    <addaction name="remove_region_action"/>
    <addaction name="plot_options_action"/>
    <addaction name="export_plot_action"/>
+   <addaction name="change_unit_action"/>
```

Comment:
Are these unit dialog ui changes different from the ones in your other PR?

Reply:
Nope